src/endpoints/authentication/impl.cpp

```diff
@@ -388,16 +377,20 @@ namespace irods::http::handler
 					constexpr auto basic_auth_scheme_prefix_size = 6;
 					const auto [username, password]{decode_username_and_password(iter->value().substr(pos + basic_auth_scheme_prefix_size))};
 
+                    if (username.length() == 0 || password.length() == 0) {
```

Comment:
do we do length checks like this?   or do we have an `.empty()` method?

Reply:
Forgot about `empty()`...